libraries/AP_FETtecOneWire/AP_FETtecOneWire.h

```diff
@@ -56,6 +56,8 @@ class AP_FETtecOneWire : public AP_ESC_Telem_Backend
     static AP_FETtecOneWire *_singleton;
     bool _initialised;
     AP_HAL::UARTDriver *_uart;
+    bool use_hdplex = false;

```

Comment:
```suggestion
    bool _use_hdplex;
```
member variable names must be prefixed with_
boolean member variables default to false, so no need to init them.

Reply:
I fixed it myself.